### PR TITLE
fix: dia-1301 : adding .1 sec delay for kafka message injection.

### PR DESCRIPTION
**Pull request by forum-hs**
Adding delay as temporary fix

### CODE REVIEW

**Comment by robot-ci-heartex** [comment]
Commit https://github.com/HumanSignal/infra/commit/bbbc5c927e3bc1b986eb3f98ac341a2c27d71d79 is created.

**Review by matt-bernstein — APPROVED** [review]

File: server/app.py [hunk 1]
```diff
@@ -177,6 +183,7 @@ async def submit_batch(batch: BatchData):
     try:
         for record in batch.data:
             await producer.send_and_wait(topic, value=record)
+            time.sleep(0.1)
```
Comment (pakelley) [inline server/app.py:186]: sorry, can we just leave a comment explaining this is a temp fix? ideally with a loud FIXME in front

**Review by pakelley — APPROVED** [review]

**Comment by robot-ci-heartex** [comment]
Commit https://github.com/HumanSignal/infra/commit/963076a3f55f215e936104ad5ae9992e5fc8b552 is created.

**Comment by robot-ci-heartex** [comment]
Commit https://github.com/HumanSignal/infra/commit/61c4a11ea512ef7ec6016aedbb9ccb1f43130eaf is created.

**Pull request merged**